src/main/java/com/locationinformationmicroservice/controller/LocationInformationController.java

```diff
@@ -14,7 +14,7 @@ public class LocationInformationController {
   
   @GetMapping("/information")
   @ResponseBody
-  public HashMap<String,String> location_information(@RequestParam String lat, @RequestParam String lon, @RequestParam String image_api_key, @RequestParam String geo_api_key) {    
+  public HashMap<String,Object> location_information(@RequestParam String lat, @RequestParam String lon, @RequestParam String image_api_key, @RequestParam String geo_api_key) {    
```

Comment:
The other way I found was to make classes for different types. This is simpler and works, so I am happy.